lib/rules/indent.js

```diff
@@ -375,12 +376,25 @@ module.exports = {
                 lastToken = sourceCode.getTokenBefore(lastToken);
             }
 
+            const textBeforeClosingParenthesis = sourceCode.getText(lastToken, lastToken.loc.start.column).slice(0, -1);
+            const indentChar = indentType === "space" ? " " : "\t";
+            const replaceRegExp = new RegExp(indentChar, "g");
+            const textLenWithoutIndentation = textBeforeClosingParenthesis.replace(replaceRegExp, "").length;
```

Comment:
Since we only care about whether `textBeforeClosingParenthesis` contains any non-whitespace characters, I think this logic can be simplified to just compare `textBeforeClosingParenthesis.trim()` to the empty string. For example:

```js
if (textBeforeClosingParenthesis.trim()) {

  // There are tokens before the closing paren, don't report this case
  return;
}
```